### ISSUE TITLE
General Model of Mathematics Part 1 and want to learn more mathematics

### ISSUE THREAD

**Issue by kamiloze2004** [leios/SoME_Topics#38]
This will be **our videos** is for **our Noble Prize**, and I will make **General Model of Mathematics** for **_ Our Noble Prize_** and **it will push human knowledge to places no one can expanding to go.** deathkamiloze@gmail.com 
What is format of General Model video, That is way to basic for me, and I will change the world with my ideas.

## Target topic categories

I am working on _**General Model of Mathematics**_ because I want to know more mathematical equation to explain the universe, plus I heard every galaxy, solar system, and planet having different laws of physic like theory of everything never work every place in the universe... **Who want to work a video is about General Model of Mathematics?**  _Our understanding higher dimension beyond 3D is wrong_, and I think the solution and equation is complete different but it is in a matrix (**(x,y) Matrix + (x,y,z) matrix = (x,(y(1),y(2)),z) matrix** and _how to make a prefect functional dimension matrix plus how to calculating it?_) **complete beyond our modern mathematic**. **What is connection between Quantum Forces, Fields, and Interactions Theory aka, what is Quantum Model Theory?**

## Target medium

A video series, and **Noble Prize**...

![Photo on 09-06-2022 at 20 20](https://user-images.githubusercontent.com/102604792/172940286-9d7395ba-3ca4-4cb1-b864-ce9e1d9853fb.jpg)

**Comment by leios** [comment]
@kamiloze2004 It seems that you already opened #37 . Would you mind if I closed that one and edited this one slightly to reflect what you want?

**Comment by kamiloze2004** [comment]
> @kamiloze2004 It seems that you already opened #37 . Would you mind if I closed that one and edited this one slightly to reflect what you want?
Yes, I want that.

**Comment by leios** [comment]
Ok, I'll edit this post. What is the best way to contact you? Discord? Email?

**Comment by leios** [comment]
@kamiloze2004 did you intend to close this issue?

**Comment by kamiloze2004** [comment]
> @kamiloze2004 did you intend to close this issue?

Oops, It maybe cosmic radiation made it plus I didn't closed the issue.

**Comment by kamiloze2004** [comment]
I am putting it in my bookmarks because I am have huge term for around 25-75 videos on the date of being to end the Summer of Math Exposition number 3 in the series, plus talking about non-mathematical topics and mathematics topics plus combining the topic into few last videos and then great finally in the last video.

**Comment by kamiloze2004** [comment]
@3b1b- Please, get more people in this term.

**Comment by leios** [comment]
@kamiloze2004 Please refrain from advertising your topic on other collaboration requests.

Also: please feel free to add more detail about your ideas here! Papers, general ideas, etc. go a long way!

**Comment by leios** [comment]
Hey, please keep the discussion about this issue *on the issue itself*. Do not mention this topic on other topics unless it is directly related.

If you keep spamming other topics, I will have to ban you from this repository

**Comment by kamiloze2004** [comment]
I will keep talk in this topic, and I will talk about their topic.

**Comment by kamiloze2004** [comment]
> Hey, please keep the discussion about this issue _on the issue itself_. Do not mention this topic on other topics unless it is directly related.
> 
> If you keep spamming other topics, I will have to ban you from this repository

Sorry, I will stop.

**Comment by leios** [comment]
I will not delete your posts right now, but you must explain in detail how your topic is related to:
1. Symmetry breaking
2. Fractional dimensions (via simple addition of 2+3D = 2.5D)
3. Fractal geometry
4. Quantum computation
5. The Krystal Ratio

If you cannot do this, then I will need to delete the relevant posts.

**Comment by kamiloze2004** [comment]
I don't have any paper, but I have 32.5-33 page of equations and I am only 17 years old.

**Comment by leios** [comment]
Oh, that's impressive! Could you share bits of that here? Not all of it, but some details, like what, exactly, you are modelling? How are you generalizing mathematics, exactly?

**Comment by kamiloze2004** [comment]
Complex Formulas like General Model of Particle Physic equation, and I give more flexibility than Standed Model of Particle Physic!

**Comment by kamiloze2004** [comment]
![Photo on 09-06-2022 at 20 20](https://user-images.githubusercontent.com/102604792/172917548-b2c5aa21-cf2b-4d77-bed3-3919b613616b.jpg), That is first page of equations

**Comment by leios** [comment]
Oh man, that's dense.

Maybe it makes sense for SoME2 to be your chance to compress everything into an understandable format? Honestly, you have all summer, so if you can do that in the next few weeks, you still have plenty of time to find a collaborator for SoME2. Otherwise, I think this might be a great chance to run solo! We allow written entries, so if you end up writing a textbook by the end of the summer, that would be really impressive.

**Comment by ajayberriman** [comment]
I am inspired but far too intimidated by your obvious intelligence to feel I can help you.

Is the Krystal ratio even relevant to your understanding of the universe or have you transcended such numerical concerns?

**Comment by kamiloze2004** [comment]
I can not because I have disabilities like young Albert Einstein (From his early life), and I need people do for me because I am delayed with languages (I don't write at all to the age of 8 age and I don't write at all to the age of 10 age).

**Comment by kamiloze2004** [comment]
> I am inspired but far too intimidated by your obvious intelligence to feel I can help you.
> 
> Is the Krystal ratio even relevant to your understanding of the universe or have you transcended such numerical concerns?

I will help you to from complex beautiful Krystal Ratio Matrix (KRM), what type of person in this term do you want to be?

**Comment by kamiloze2004** [comment]
![FUuJM5TXoAcM0zi](https://user-images.githubusercontent.com/102604792/172922526-a3659a0b-328a-43ed-b1e6-ce18ec9c4bd4.jpeg) - I made it, and I did it for fun.

**Comment by kamiloze2004** [comment]
> I will not delete your posts right now, but you must explain in detail how your topic is related to:
> 
> 1. Symmetry breaking
> 2. Fractional dimensions (via simple addition of 2+3D = 2.5D)
> 3. Fractal geometry
> 4. Quantum computation
> 5. The Krystal Ratio
> 
> If you cannot do this, then I will need to delete the relevant posts.

They are combine into Dimensionality Mathematic, and it is new type of mathematic will help other topic can go beyond their subjects within Mathematical Generality.

**Comment by ajayberriman** [comment]
Wow!

I suppose if you really need the help...

I've always thought I'd make a good Head of KRM. I won't do anything that goes against the Prime Directive, though.

**Comment by kamiloze2004** [comment]
I love Star Trek Jokes, and USS-Mathematicttia will be flying be this year or next year.

**Comment by kamiloze2004** [comment]
First solve this- Г(x,y,z)+ Г(x,y)= what it look like?

**Comment by kamiloze2004** [comment]
> Wow!
> 
> I suppose if you really need the help...
> 
> I've always thought I'd make a good Head of KRM. I won't do anything that goes against the Prime Directive, though.

First solve this-Г(t[x^t, y^t ,z^t])^Г(c[t^c] ,c[a^c], c[b^c]) [Г(x,y,z)^Г(t,a,b)+ Г(x,y)^Г(t,a)]= what it look like, it is dimensionality problem of 7 function dimension gamma problem- one most easiest problem of dimensionality machines.

**Comment by ajayberriman** [comment]
I'm glad we share an interest.

Remember that I'm not an expert but I do believe the solution you are looking for is

$$x = 6 , y = 6 , z = 6$$

but there could be others - I didn't carry the 6 for a start.

**Comment by kamiloze2004** [comment]
Close, the solution will x=#,y=#,z=#,t=#,a=#,b=#,c=#,Г=matrix of all 7 dimension solution by it is the way of j-function 1st Harmonic Universe matrix ^ j-function 2nd Harmonic Universe matrix time j-function Harmonic 7/3 Universe matrix.

**Comment by kamiloze2004** [comment]
> I'm glad we share an interest.
> 
> Remember that I'm not an expert but I do believe the solution you are looking for is
> 
> x=6,y=6,z=6
> 
> but there could be others - I didn't carry the 6 for a start.

Close, the solution will x=6,y=6,z=6,t=#,a=#,b=#,c=#,Г=matrix of all 7 dimension solution by it is the way of j-function 1st Harmonic Universe matrix ^ j-function 2nd Harmonic Universe matrix time j-function 7/3th Harmonic Universe matrix.

**Comment by kamiloze2004** [comment]
> You know what, I've gone and ignored 4 spatial dimensions! I can't believe I did that.
> 
> I'll do better.

Read this book "Proposing "NEW" Cosmology Beyond Science," by Theresa Talea.

**Comment by kamiloze2004** [comment]
> You know what, I've gone and ignored 4 spatial dimensions! I can't believe I did that.
> 
> I'll do better.

Read this book "Proposing "NEW" Cosmology Beyond Science," by Theresa Talea.

**Comment by kamiloze2004** [comment]
> You know what, I've gone and ignored 4 spatial dimensions! I can't believe I did that.
> 
> I'll do better.

deathkamiloze@gmail.com

**Comment by sv-cmyk** [comment]
Dude, you can't just mention yourself in 8 different Seekings for no apparent reason or association. Please write a comment that illustrates the relationships from the posts to yours.

**Comment by oauo** [comment]
When not mentioning yourself you aren't even adding anything useful. You sound like you want to be seen as a genius but all you've shown to prove it is an image that couldn't have been taken worse and is so messy I can't begin to read any of it.

**Comment by leios** [comment]
Hey @kamiloze2004 As you still have not shown the relationship between your post and all the others, I will need to delete your other posts.

Unfortunately, if this happens again, I will need to ban you from the repository.

I think it would be worthwhile to:
1. think deeply about what you are trying to explain
2. explain it in a new issue / topic. This one is a bit messy and hard to follow due to the spam in other topics.

**Comment by kamiloze2004** [comment]
> Dude, you can't just mention yourself in 8 different Seekings for no apparent reason or association. Please write a comment that illustrates the relationships from the posts to yours.

It is Mathematical Unification of GMM (General Model of Mathematics

**Comment by kamiloze2004** [comment]
> When not mentioning yourself you aren't even adding anything useful. You sound like you want to be seen as a genius but all you've shown to prove it is an image that couldn't have been taken worse and is so messy I can't begin to read any of it.

You need grass, and it is normal you have that reaction but you are a person try to stop history going somewhere.
